server/channels/store/searchtest/helper.go

```diff
@@ -301,7 +301,7 @@ func (th *SearchTestHelper) createDirectChannel(teamID, name, displayName string
 }
 
 func (th *SearchTestHelper) createGroupChannel(teamID, displayName string, users []*model.User) (*model.Channel, error) {
-	userIDS := make([]string, len(users))
```

Comment:
Huh, this was just wrong! I guess the empty strings git filtered out somewhere by luck?

Reply:
Btw, this is coming from a new linter called `makezero`. It's very effective because I ran into these cases myself.